evidence/alin/0001-alin-promote-II-2024.md

```diff
@@ -0,0 +1,48 @@
+# Evidence-0001: Promotion to Rank II
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2024/06/21)                                                             |
+| **Submitted by**| Alin Dima                                                                                   |
+
+
+## Member details
+
+- Matrix username: `@alin:parity.io`
+- Polkadot address: `148f8D1P4CP2tV8JuaVHzEXQQgj3jBxEg3k9qZydPzkJjbQG`
+- Current rank: I
+- Date of initial induction: 2024-04-26
+- Date of last report: N/A
+- Area(s) of Expertise/Interest: Parachains consensus
+
+
+## Reporting period
+
+- Start date: 2024/04/26
```

Comment:
Alin was [promoted to Rank 1](https://collectives.polkassembly.io/referenda/92) on 22nd April 2024.
Since this happened less that 12 months ago, a promotion to Rank 2 is not currently possible. 
For now, please update this form to request a retention at Rank 1.